src/vsr/grid.zig

```diff
@@ -1024,7 +1024,7 @@ pub fn GridType(comptime Storage: type) type {
             if (result != .valid) {
                 const header =
                     mem.bytesAsValue(vsr.Header.Block, block.*[0..@sizeOf(vsr.Header)]);
-                log.err(
```

Comment:
same comment as for latent sector errors -- although technically this is a recoverable failure that warrants a warn, this is _very_ curious, and perhaps is worth an .err just for that!